documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -13,6 +13,7 @@
 * In addition to node_modules, bin, build, .git, .gradle, out, packages, target, the Gradle wrapper directory `gradle` will be excluded from signature scan by default. Use
   [detect.excluded.directories.defaults.disabled](properties/configuration/paths.md#detect-excluded-directories-defaults-disabled-advanced) to disable these defaults.
 * Support for Gradle is now extended to Gradle 8.2.
+* Removed dependency on [solution_name] dependencies for init-detect.gradle script. This will prevent [solution_name]'s dependencies from being included in the dependency verification by Gradle.
```

Comment:
Point comes across so going to approve, feel free to ignore if you think this a nit pick. Seems like variants of the word dependency are used a lot. Maybe change the first sentence to say something like Removed reliance on [solution_name]'s libraries for init-detect.gradle script.

Reply:
Done, thanks.